src/mapbox/layers/TrajservLayer.js

```diff
@@ -108,9 +108,10 @@ class TrajservLayer extends mixin(TrackerLayer) {
    * @private
    */
   onMove() {
-    this.map
-      .getSource('canvas-source')
-      .setCoordinates(getSourceCoordinates(this.map));
+    this.map.getSource('canvas-source').coordinates = getSourceCoordinates(
```

Comment:
this can't be 'canvas-source' 
```suggestion
    this.map.getSource(this.key).coordinates = getSourceCoordinates(
```